skcosmo/SparseMethods/Sparsified.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from abc import ABCMeta
+from sklearn.utils import check_array
+from sklearn.utils.validation import check_is_fitted
+from sklearn.base import BaseEstimator, TransformerMixin, RegressorMixin
+from sklearn.metrics.pairwise import pairwise_kernels
+from ..preprocessing.flexible_scaler import KernelFlexibleCenterer
+from ..selection.FPS import SampleFPS
+
+
+class _Sparsified(TransformerMixin, RegressorMixin, BaseEstimator, metaclass=ABCMeta):
+    """
+    Super-class defined sparcified methods
+
+        :param kernel: the kernel used for this learning
+        :param gamma: exponential factor of the rbf and sigmoid kernel
+        :param degree: polynomial kernel degree
+        :param coef0: free term of the polynomial and sigmoid kernel
+        :param kernel_params: kernel parameter set
+        :param n_active: the size of the small dataset used in learning
+        :param regularization: regularization parameter for SparseKRR and SparseKRCovR
+        :param tol: Relative accuracy for eigenvalues (stopping criterion) The default value of 0 implies machine precision.
```

Comment:
Everything that is in the description should be in the constructor. Is this required for this class?